README.md

```diff
@@ -1,158 +1,236 @@
-az group create --name "<YourResourceGroupName>" --location "East US"
+az group create --name "<MyResourceGroupName>" --location eastus
 ```
 
-The resource group you just created is used throughout this tutorial.
+The Resource Group you just created is used throughout this tutorial.
 
 ## Create an Azure Key Vault
 
-Next you create a Key Vault using the resource group created in the previous step. Although “ContosoKeyVault” is used as the name for the Key Vault throughout this article, you have to use a unique name. Provide the following information:
+Next you will create a Key Vault using the Resource Group created in the previous step. Provide the following information:
 
-* Vault name - **Select a Key Vault Name here**.
-* Resource group name - **Select a Resource Group Name here**.
-* The location - **East US**.
+* Vault name - Create a custom name and replace `"<MyKeyVaulName>"` below.
+* Resource group name - Use the same Resource Group Name you used above.
+* The location - Use the same location that you created the Resource Group in above.
 
 ```azurecli
-az keyvault create --name "<YourKeyVaultName>" --resource-group "<YourResourceGroupName>" --location "East US"
+az keyvault create --name "<MyKeyVaultName>" --resource-group "<MyResourceGroupName>" --location eastus
 ```
 
-At this point, your Azure account is the only one authorized to perform any operations on this new vault.
-
-## Add a secret to key vault
+## Add a Secret to Key Vault
 
-We're adding a secret to help illustrate how this works. You could be storing a SQL connection string or any other information that you need to keep securely but make available to your application. In this tutorial, the password will be called **AppSecret** and will store the value of **MySecret** in it.
+Next, we'll add a secret to KeyVault to help illustrate how Secret Value works. You could store a SQL connection string or any other information that you need to keep secure and make it available to your application. 
 
-Type the commands below to create a secret in Key Vault called **AppSecret** that will store the value **MySecret**:
+In this tutorial, the password will be called **AppSecret** and will store the value of **MySecret** in it:
 
 ```azurecli
-az keyvault secret set --vault-name "<YourKeyVaultName>" --name "AppSecret" --value "MySecret"
+az keyvault secret set --vault-name "<MyKeyVaultName>" --name AppSecret --value MySecret
 ```
 
-To view the value contained in the secret as plain text:
+To view the value contained in the Secret as plain text, please type the following command. This command shows the Secret Information including the URI. After completing these steps, you should have a URI to a Secret in an Azure Key Vault. Copy the output from the previous command to text editor. You will need it later:
 
 ```azurecli
-az keyvault secret show --name "AppSecret" --vault-name "<YourKeyVaultName>"
+az keyvault secret show --name AppSecret --vault-name "<MyKeyVaultName>"
```

Comment:
Sample placeholder comment.

Reply:
AppSecret is a fixed term here. Not a sample name.